README.md

```diff
@@ -1,30 +1,20 @@
-# Literature Reviews
+# Code Confusion Catalog
 
-This repo houses markdown documents exploring proposed language and library additions to the Elm language. This includes:
+This repo catalogs painful and confusing interactions with [Elm](http://elm-lang.org/).
+We do this to distill a whole lot of pain down to a handful of solutions.
 
-- Extensions to the Elm language, syntax, and semantics.
-- Modifications or extensions of the core Elm packages. These are for the Elm "standard library", including packages like `List`, `Dict`, and `Json.Decode`.
-- Large community-focused informational documents (why do we do it *that* way?)
+If that sounds weird to you, watch [*Code is the Easy Part* by Evan Czaplicki](https://www.youtube.com/watch?v=DSjbTC-hvqQ&list=PLglJM3BYAMPH2zuz1nbKHQyeawE4SN0Cd&index=1) for more context.
 
-## Gathering feedback before submitting
+## Should I Open an Issue?
 
-It's often helpful to get feedback on your concept before getting into the detail required for a literature review.
-You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request. In addition you can post a message to the [elm-discuss mailing list](https://groups.google.com/forum/#!forum/elm-discuss) or chat to the folks over at [Slack](https://elmlang.herokuapp.com/).
+If you're looking for help, you're probably better off asking on [elm-discuss](https://groups.google.com/forum/#!forum/elm-discuss), [Reddit](https://www.reddit.com/r/elm/), or in the #beginners or #general channel on [the Elm Slack](http://elmlang.herokuapp.com/).
```

Comment:
In the interest of pushing people away from posting help on elm-discuss, let's remove that from here.